component/auth/register.jsx

```diff
@@ -27,11 +27,9 @@ function RegisterPage() {
     await httpService
       .post(`/api/user`, values)
       .then(({ data: response }) => {
-        if (response.status === 200) {
-          openNotificationBox("success", response.message, 3);
-          registerForm.resetFields();
-          router.push("/auth/login");
-        }
+        openNotificationBox("success", response.message, 3);
+        registerForm.resetFields();
+        router.push("/auth/login");
         setLoading(false);
```

Comment:
Set loading you can do in finnaly block as you may have to repeat  the setloading false in case of exception